app/commands/submission/test_run/init.rb

```diff
@@ -43,15 +43,8 @@ def call
       delegate :solution, to: :submission
 
       def exercise_filepaths
-        exercise_repo.tooling_filepaths.select do |filepath|
-          # Skip non-functional files
-          next false if filepath.starts_with?(".docs")
-          next false if filepath == "README.md"
-
-          # Skip submitted files
-          next false if submission.valid_filepaths.include?(filepath)
-
-          true
+        exercise_repo.tooling_filepaths.reject do |filepath|
```

Comment:
I've tried moving this functionality to submission, as it is the same here and for the analyzer, but didn't do that as the `exercise_repo` used here is not necessarily the same as the submission exercise's git repo (due to the test run being allowed to pass in the `git_sha`).